src/store/modules/invites.js

```diff
@@ -13,16 +13,13 @@ export default {
     },
   },
   actions: {
-    async claim({ rootState: { current, sdk }, rootGetters: { account } }, secretKey) {
+    async claim({ rootState: { sdk }, rootGetters: { account } }, secretKey) {
       const publicKey = Crypto.getAddressFromPriv(secretKey);
-      // TODO: Remove this after fixing https://github.com/aeternity/aepp-sdk-js/issues/1261
-      const { name, instance } = sdk.pool.get(current.network);
-      // TODO: Remove this after merging https://github.com/aeternity/aepp-sdk-js/pull/1060
-      const s = await Universal({
-        nodes: [{ name, instance }],
-        accounts: [MemoryAccount({ keypair: { publicKey, secretKey } })],
-      });
-      await s.transferFunds(1, account.address, { payload: 'referral', verify: false });
+      await sdk.transferFunds(
+        1,
```

Comment:
It would be good to add constant for this value, people might get confused what does number 1 means

Reply:
Ideally, they would check sdk documentation. This code shouldn't document sdk methods.